continuousintegration.py

```diff
@@ -24,7 +24,9 @@ def watch_ci():
     s.listen(10)
     print 'listening for CI changes'
 
-    while 1:
+    api_url = 'https://api.github.com/repos/Charcoal-SE/SmokeDetector/'
```

Comment:
I think it looks better the original way, since it's only two links I don't think we need to factor out the API URL into a variable